File: growingPlant/growing_plant.java
int growingPlant(int upSpeed, int downSpeed, int desiredHeight) {
    
    int days = desiredHeight / (upSpeed - downSpeed);
    int a = 0;
    int n = 0;
    for(int i = 0; i < days; i++){
        n = i;
        System.out.println(a);
        a += upSpeed - downSpeed;
        if (a+downSpeed >= desiredHeight || a>=desiredHeight) break;
    }
    return n+1;
}

File: growingPlant/growing_plant.js
function growingPlant(upSpeed, downSpeed, desiredHeight) {

    let days = Math.round(desiredHeight / (upSpeed - downSpeed));
    let a = 0;
    for (let i = 1; i <=days; i++){
        a += upSpeed - downSpeed;
        if(a+downSpeed >= desiredHeight || a >= desiredHeight){
            return i;
        }

    }
}
